web/shapeworks/src/components/ShapeViewer/methods.js

```diff
@@ -241,128 +240,99 @@ export default {
+                  orientations.push([dx, dy, dz])
+                  colors.push(d / 10 + 0.5)
                 }
-            }
-
-            vectorSource.getPointData().addArray(
+              }
+      
+              vectorSource.getPointData().addArray(
                 vtkDataArray.newInstance({
-                    name: 'color',
-                    values: colors
+                  name: 'color',
+                  values: colors
                 })
-            )
-            vectorSource.getPointData().addArray(
+              )
+              vectorSource.getPointData().addArray(
                 vtkDataArray.newInstance({
-                    name: 'normal',
-                    values: orientations.flat(),
-                    numberOfComponents: 3
+                  name: 'normal',
+                  values: orientations.flat(),
+                  numberOfComponents: 3
                 })
-            )
-            vectorSource.getPoints().setData(locations.flat(), 3)
-            vectorSource.getVerts().setData(verts);
-            vectorActor.setMapper(vectorMapper)
-            vectorMapper.setInputData(vectorSource)
-            vectorMapper.addInputConnection(vectorShape.getOutputPort(), 1)
-            vectorMapper.setScaleFactor(5)
-            vectorMapper.setColorByArrayName('color')
-            vectorMapper.setOrientationArray('normal')
-            vectorMapper.setLookupTable(this.lookupTable)
-            renderer.addActor(vectorActor)
-        }
-
-        const colorArray = vtkDataArray.newInstance({
-            name: 'color',
-            values: colorValues
-        })
-        mapper.getInputData().getPointData().addArray(colorArray)
-        mapper.getInputData().modified()
-        mapper.setLookupTable(this.lookupTable)
-        mapper.setColorByArrayName('color')
-
-        this.prepareColorScale()
-
-        this.render()
-    },
+              )
+              vectorSource.getPoints().setData(locations.flat(), 3)
+              vectorSource.getVerts().setData(verts);
+              vectorActor.setMapper(vectorMapper)
+              vectorMapper.setInputData(vectorSource)
+              vectorMapper.addInputConnection(vectorShape.getOutputPort(), 1)
+              vectorMapper.setScaleFactor(5)
+              vectorMapper.setColorByArrayName('color')
+              vectorMapper.setOrientationArray('normal')
+              vectorMapper.setLookupTable(this.lookupTable)
+              renderer.addActor(vectorActor)
+            }
+      
+            const colorArray = vtkDataArray.newInstance({
+              name: 'color',
+              values: colorValues
+            })
+            mapper.getInputData().getPointData().addArray(colorArray)
+            mapper.getInputData().modified()
+            mapper.setLookupTable(this.lookupTable)
+            mapper.setColorByArrayName('color')
+      
+            this.prepareColorScale()
+      
+            this.render()
+          },
```

Comment:
Very minor, but could you check all these whitespace changes? Is it supposed to be aligned like this?

Reply:
And as a final thing to fix, be sure to fix anything the CI tests catch. Then I think you're good to merge.